product_docs/docs/eprs/7/07_common_operations/11_using_ssl_connections.mdx

```diff
@@ -350,22 +374,86 @@ To set up different source and target database types (for example, source databa
 1.  Generate the certificate for `POSTGRESQL` database and follow the table for placing certificate files in the default directory.
 2.  Copy these certificates in the EDB Postgres Advanced Server data directory.
 
-```text
-[root@localhost data]# cp /var/lib/pgsql/11/data/root.crt .
-[root@localhost data]# cp /var/lib/pgsql/11/data/server.crt .
-[root@localhost data]# cp /var/lib/pgsql/11/data/server.key .
-```
+    ```text
+    cd /var/lib/edb/as14/data
+    [root@localhost data]# cp /var/lib/pgsql/14/data/root.crt .
+    [root@localhost data]# cp /var/lib/pgsql/14/data/server.crt .
+    [root@localhost data]# cp /var/lib/pgsql/14/data/server.key .
+    ```
 
 1.  Use the following commands to change the permissions of the certificates in the EDB Postgres Advanced Server data directory.
 
-```text
-[root@localhost data]# sudo chown enterprisedb root.crt server.crt server.key
-[root@localhost data]# sudo chgrp enterprisedb root.crt server.crt server.key
-[root@localhost data]# sudo chmod 600 root.crt server.crt server.key
-```
+    ```text
+    [root@localhost data]# sudo chown enterprisedb root.crt server.crt server.key
+    [root@localhost data]# sudo chgrp enterprisedb root.crt server.crt server.key
+    [root@localhost data]# sudo chmod 600 root.crt server.crt server.key
+    ```
 
 <div id="request_ssl_to_xdb_server_dbs" class="registered_link"></div>
 
+## Using different databases for the source and target 
+
+Follow these steps if you are using different databases for the source and target; for example, if you are using PostgreSQL for your source database and EDB Postgres Advanced Server for your target database.
+
+!!!note
+The commands in this section assume `CN=db user name`.
+!!!
+
+1. Generate the certificate for the PostgreSQL database. See [Configuring SSL on a Postgres database server](#configuring-ssl-on-a-postgres-database-server).
+
+1. Configure SSL for Replication Server. See the steps for PostgreSQL in [Configuring SSL for the publication server and subscription server](#configuring-ssl-for-the-publication-server-and-subscription-server).
```

Comment:
The serial number of this command should be 2. The numbering currently is 1 for all steps. It should be 1 to 7.

Reply:
Markdown automatically orders numbered lists. You do not need to number them manually. If you look at the test deployment, you'll see the lists are ordered correctly.

Reply:
The numbering is correct on the page.